src/Input.tsx

```diff
@@ -75,7 +76,7 @@ export function GraphInput<S extends Schema, K extends keyof S>(
 				dominantBaseline="middle"
 				fontSize={fontSize}
 			>
-				{props.input}
+				{kinds[node.kind].inputs[input].label}
```

Comment:
Nit: Maybe this could be done in the parent component? But not really necessary at the moment. Let me know what you think.